modules/nf-core/stringtie/merge/tests/main.nf.test

```diff
@@ -0,0 +1,82 @@
+nextflow_process {
+
+    name "Test Process STRINGTIE_MERGE"
+    script "../main.nf"
+    process "STRINGTIE_MERGE"
+    tag "modules"
+    tag "modules_nfcore"
+    tag "stringtie"
+    tag "stringtie/merge"
+
+    test("homo_sapiens - forward strandedness") {
+
+        setup {
+            run("STRINGTIE_STRINGTIE") {
+                script "../../stringtie/main.nf"
+                process {
+                """
+                input[0] = [
+                                [ id:'test', strandedness:'forward' ], // meta map
+                                [ file(params.test_data['homo_sapiens']['illumina']['test_paired_end_sorted_bam'], checkIfExists: true) ]
+                            ]
+                input[1] = file(params.test_data['homo_sapiens']['genome']['genome_gtf'], checkIfExists: true)
+                """
+            }
+            }
+
+        }
+
+        when {
+            process {
+                """
+                input[0] = STRINGTIE_STRINGTIE.out.transcript_gtf.map { it -> it[1] }
+                input[1] = file(params.test_data['homo_sapiens']['genome']['genome_gtf'], checkIfExists: true)
+                """
+            }
+        }
+
+        then {
+            assertAll(
+            { assert process.success },
+            { assert snapshot(process.out.gtf).match("fs_gtf") },
+            { assert snapshot(process.out.versions).match("fs_versions") }
```

Comment:
I think @mashehu preferred to just snapshot `process.out` to be more concise.

Reply:
there are few emits that are optional outputs...so cannot do `process.out`